### PR TITLE
[DO-1254] Improve format date function

### PR DESCRIPTION
**Pull request by Iva-khar**
The format date function returns error if invalid input now. It should return nothing in that case.

### CODE REVIEW

File: data_ocean/utils.py [hunk 1]
```diff
@@ -58,11 +58,14 @@ def cut_first_word(string):
...
-        str_ddmmyy = str_ddmmyy.replace("/", ".")
-    return datetime.datetime.strptime(str_ddmmyy, "%d.%m.%Y").strftime("%Y-%m-%d")
+    if str_ddmmyy:
+        if ';' in str_ddmmyy:
+            str_ddmmyy = str_ddmmyy.replace(";", "")
+        if '/' in str_ddmmyy:
+            str_ddmmyy = str_ddmmyy.replace("/", ".")
+        return datetime.datetime.strptime(str_ddmmyy, "%d.%m.%Y").strftime("%Y-%m-%d")
+    else:
+        return
```
Comment (roman-tiukh) [inline data_ocean/utils.py:68]: else return?

File: data_ocean/utils.py [hunk 2]
```diff
@@ -58,11 +58,14 @@ def cut_first_word(string):
 
 # deleting ';' from a string, then converting it to the datetime format 'Y-m-d'
 def format_date_to_yymmdd(str_ddmmyy):
-    if ';' in str_ddmmyy:
-        str_ddmmyy = str_ddmmyy.replace(";", "")
-    if '/' in str_ddmmyy:
-        str_ddmmyy = str_ddmmyy.replace("/", ".")
-    return datetime.datetime.strptime(str_ddmmyy, "%d.%m.%Y").strftime("%Y-%m-%d")
+    if str_ddmmyy:
```
Comment (IegorSoboliev) [inline data_ocean/utils.py:61]: Maybe, it is better to make such a check before calling that function?

File: data_ocean/utils.py (hunk 1 above)
Comment (Iva-khar) [reply data_ocean/utils.py:68]: It used in in like this callings:
```
creditor_reg_end_date = format_date_to_yymmdd(
                record.xpath('TERMINATION_STARTED_INFO/CREDITOR_REQ_END_DATE')[0].text) or '1990-01-01'
```
It's error when invalid input instead '1990-01-01'

Comment (Iva-khar) [reply data_ocean/utils.py:68]: I need just return

File: data_ocean/utils.py (hunk 2 above)
Comment (Iva-khar) [reply data_ocean/utils.py:61]: Many checkings in different places vs one check in function? Disagree

Comment (IegorSoboliev) [reply data_ocean/utils.py:61]: Hope sometimes we do have data without None values)

File: data_ocean/utils.py (hunk 1 above)
Comment (Iva-khar) [reply data_ocean/utils.py:68]: If you mean that 
```
else:
    return 
```
is excess?

Comment (roman-tiukh) [reply data_ocean/utils.py:68]: yes

Comment (Iva-khar) [reply data_ocean/utils.py]: done

File: data_ocean/utils.py (hunk 2 above)
Comment (Iva-khar) [reply data_ocean/utils.py:61]: )

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**